requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests>=2.14.0
 mock==1.3.0
 pre-commit
+pyjwt[crypto]>=1.7.1
```

Comment:
the recommended way of requiring this dependency, since it includes "crypto" and is easier to tell both are used "together"